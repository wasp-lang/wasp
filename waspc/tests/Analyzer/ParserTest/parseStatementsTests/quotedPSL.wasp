test PSL {=psl
  id Int @id
psl=}
