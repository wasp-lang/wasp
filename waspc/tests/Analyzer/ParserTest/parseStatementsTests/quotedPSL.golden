(AST
  (Decl@1:1-3:5 type=test name=PSL
    (Quoter@1:10-3:5 tag=psl
      {=
        id Int @id
      =}
    )
  )
)
